src/logs/logger.ts

```diff
@@ -0,0 +1,38 @@
+import winston, { transports } from 'winston';
+// import * as fs from 'node:fs';
+// import path from 'node:path';
+
+// const logDir = '../../src/logs';
+
+// if (!fs.existsSync(logDir)) {
+//   fs.mkdirSync(logDir);
+// }
+
+const { combine, timestamp, json } = winston.format;
+const { printf, colorize, align } = winston.format;
```

Comment:
You can turn off this rule `object-curly-newline`, it does not make sense
```suggestion
const { combine, timestamp, json, printf, colorize, align } = winston.format;
```

Reply:
Done sir